### PR TITLE
faster debug

### PR DESCRIPTION
**Pull request by jwallace42**
Found faster debugging steps than the provided documentation.

### CODE REVIEW

**Comment by SteveMacenski** [comment]
Its not clear to me what about this is "faster"? Its also less generic, one intention I had for this was it was instructions you could use in any project in the ROS Ecosystem, just happens to be targeted to Nav2 for the purposes of demonstration. This would remove that and some important context of how to send parameters to these nodes in that situation.

Overall, I think its definitely more succinct, but it loses some generality I'd like to keep. 

Thoughts?

**Comment by SteveMacenski** [comment]
https://github.com/ros-planning/navigation.ros.org/commit/42da2cbea610f82506ce5a0f7eab174b4c10cbce

I think this makes that intention a bit more explicitly clear

**Comment by jwallace42** [comment]
> Its not clear to me what about this is "faster"? Its also less generic, one intention I had for this was it was instructions you could use in any project in the ROS Ecosystem, just happens to be targeted to Nav2 for the purposes of demonstration. This would remove that and some important context of how to send parameters to these nodes in that situation.
> 
> Overall, I think its definitely more succinct, but it loses some generality I'd like to keep.
> 
> Thoughts?

By "faster" I mean easier to set up. If we have to create a separate launch file that is more time consuming than just adjusting the current launch file. 

It is definitely less generic. It might be worth adding this to a separate section in the get_backtrack docs. I would like to have this in the docs to allow new contributors to debug quickly without having to worry about other launch files.

File: tutorials/docs/get_backtrace.rst [hunk 1]
```diff
@@ -170,7 +170,7 @@ Then you can deduce why it crashed.
 When you are done with GDB, type ``quit`` and it will exit the session and kill any processes still up.
 It may ask you if you want to kill some threads at the end, say yes.
 
-From Nav2 Bringup
+From Large Project
 =================
```
Comment (SteveMacenski) [inline tutorials/docs/get_backtrace.rst:174]: The `=` length must match

File: tutorials/docs/get_backtrace.rst [hunk 2]
```diff
@@ -218,6 +218,35 @@ Then you can deduce why it crashed.
 When you are done with GDB, type ``quit`` and it will exit the session and kill any processes still up.
 It may ask you if you want to kill some threads at the end, say yes.
 
+From Nav2 Bringup
+=================
+
+To debug directly from the nav2 bringup launch files you may want to do the following:
+
+- Add ``prefix=['xterm -e gdb -ex run --args']`` to the non-composed node.
```
Comment (SteveMacenski) [inline tutorials/docs/get_backtrace.rst:226]:
```suggestion
- Add ``prefix=['xterm -e gdb -ex run --args']`` to the non-composed node in the appropriate launch file. 
```

File: tutorials/docs/get_backtrace.rst (hunk 1 above)
Comment (SteveMacenski) [inline tutorials/docs/get_backtrace.rst:174]:
```suggestion
==================
```

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**